---
title: RNA Epigenetics
---


| usegalaxy.eu |
|:--------:|
| <FlatShield label="Downsampling to reduce coverage bias" message="run" href="https://usegalaxy.eu/u/wolfgang-maier/w/sars-cov-2-assigned-ont-reads-downsampling-and-coverage-analysis" alt="Galaxy workflow" /> <FlatShield label="Nanocompore-based modification detection" message="run" href="https://usegalaxy.eu/u/milad/w/ont-drs-nanocompore-methylation-collection" alt="Galaxy workflow" /> <FlatShield label="Tombo-based modification detection" message="run" href="https://usegalaxy.eu/u/milad/w/sars-cov-2-ont-tombo-level-compare" alt="Galaxy workflow" /> |
|  |


## Methylation analysis

The most reliable approach for identifying the putatively modified nucleotides from the sequencing data is by comparing the distribution of raw electrical signals between two conditions. Tombo and Nanocompore are two tools that can be used for this purpose. Here, as the alternative condition, we use the in vitro transcribed oligos of SARS-CoV-2 from Kim et al..

![A snapshot of methylation analysis workflow for analyzing DRS SARS-CoV-2 data in Galaxy.](../img/methylation-snapshot.png)




